src/stores/RolleStore.ts

```diff
@@ -84,8 +78,8 @@ export const useRolleStore: StoreDefinition<'rolleStore', RolleState, RolleGette
           administeredBySchulstrukturknoten: administrationsebene,
           rollenart: rollenArt,
           // TODO Remove casting when generator issue is fixed from the server side
```

Comment:
Besteht der Generator Fehler immer noch? Wenn ja, kann das mit gefixt werden?

Reply:
Ich glaube das ist noch pending :/

Reply:
Das Problem hier ist nicht der Enum sondern das Set. Ohne Cast macht er da ein Array draus.

Reply:
Also in diesem Scope nicht zu erledigen? Dann bitte resolven und ich approve